otc-watcher/README.md

```diff
@@ -0,0 +1,82 @@
+# otc-watcher
+
+Watches addresses on blockchain and stores output information.
+
+*NOTE*: Currently only BTC address watching is supported. Adding support for other currencies in the future will be easy, thanks to the [currency connection interface](pkg/currency/currency.go).
+
+## running
+
+Right now the only dependency is to have `btcwallet` running. Then you can start `otc-watcher` with the following command (assuming you ran `go build`):
+
+```
+./otc-watcher -rpc_node="localhost:8332" \
+              -rpc_user="username"       \
+              -rpc_pass="password"       \
+              -wallet_account="account"  \
+              -wallet_pass="password"    \
+              -port=":8080"
+```
+
+* `rpc_node` is the `btcwallet` rpc server listening address
+* `rpc_user` is the `btcwallet` rpc username
+* `rpc_pass` is the `btcwallet` rpc password
+* `wallet_account` is the `btcwallet` account name (will be created if missing)
+* `wallet_pass` is the `btcwallet` passphrase used when creating the wallet
+* `port` is the http port to listen on
+
+## http api
+
+### /register
+
+Registers an address to be watched. OTC needs to watch addresses indefinitely so there's no need for an "unregister" call right now.
+
+#### request
+
+```js
+{
+	// address to be watched
+	"address": "1Hz96kJKF2HLPGY15JWLB5m9qGNxvt8tHJ",
+	// currency type of address (must be supported by otc-watcher)
+	"currency": "BTC"
+}
+```
+
+#### response
+
+`200 OK` if good, anything else and there was an error.
+
+### /outputs
+
+Gets the outputs for an address that was previously registered.
```

Comment:
The same thing with request method here to be clear.

Reply:
Also, I think that outputs should respond with 404 in case of particular address is not added to watch list. Or to keep API simple - add address to watch list automatically when /outputs is requested for new address.